app-info/eos-extra.xml

```diff
@@ -179,20 +179,13 @@
     </categories>
   </component>
   <component type="desktop" merge="append">
-    <id>net.sourceforge.Frostwire.desktop</id>
```

Comment:
Here's the reference for Frostwire having been removed: https://phabricator.endlessm.com/T4863